.github/renovate.json

```diff
@@ -4,22 +4,32 @@
     "group:test",
     "group:linters"
   ],
-  "lockFileMaintenance": { "enabled": true },
-  "labels": ["dependencies"],
+  "lockFileMaintenance": {
+    "enabled": true
+  },
+  "labels": [
+    "dependencies"
+  ],
```

Comment:
All changes to this file seem to be unrelated to the actual issue, and there's also no separate commit explaining why they're here (they seem to have been introduced through a merge of the candidate branch). Please revert.